vedro_gitlab_reporter/_gitlab_reporter.py

```diff
@@ -74,15 +75,22 @@ def on_arg_parse(self, event: ArgParseEvent) -> None:
                            default=self._tb_show_locals,
                            help="Show local variables in the traceback output")
         group.add_argument("--gitlab-show-paths",
-                           action="store_true",
-                           default=self._show_paths,
-                           help="Show the relative path of each passed scenario")
+                           default=None,
+                           nargs="*",
+                           type=str.upper,
```

Comment:
You may consider using the `ScenarioStatus` enum as the type for better validation:

```python
group.add_argument("--gitlab-show-paths", default=None, nargs="*", type=ScenarioStatus)
```

This approach will be particularly useful for scenarios like the following:

```sh
$ vedro run --reporters gitlab --gitlab-show-paths PASSED FAILED scenarios/scenario.py
vedro run: error: argument --gitlab-show-paths: invalid ScenarioStatus value: 'scenarios/first_scenario.py'
```

In this case, the `scenarios/scenario.py` is inadvertently captured by the `*` in `--gitlab-show-paths`, leading to validation errors. Implementing the `ScenarioStatus` type will help prevent such issues by ensuring that only valid status values are accepted.